cpu-mitigations-formula/metadata/form.yml

```diff
@@ -0,0 +1,26 @@
+# This file is part of cpu-mititations-formula.
+#
+# Foobar is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Foobar is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Please no occurrences of `Foobar` in license headers anywhere (here and in other places below)! I would suggest to either fix this with the formula name or omit the license header in each file and add a license file somewhere centrally.

Reply:
So after looking at a bunch of stuff... it seems like no license stuff is at the top of the file.  they all have a license file in the root directory.  (apache 2.0)   

So, should I add an apache 2.0 license file to the root of our repo?

Reply:
Yes, @mseidl, from my perspective that sounds good. Let's do this, but maybe with a separate PR?

Reply:
sure whatever you want.